lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -143,7 +201,47 @@ export const convertToMadieMeasure = (matMeasure: MatMeasure): Measure => {
     cqlLibraryName: cqlLibraryName,
     createdBy: matMeasure.harpId,
     lastModifiedBy: matMeasure.harpId,
+    versionId: uuidv4(),
```

Comment:
Any reason for generating a new uuid versus using the measure.id value?

Reply:
@jkotanchik-SB No particular reason, just wanted to make sure it's there because it's a required field in Madie. I'll change it to measure id and remove uuid. Thanks Joe!

Reply:
@jkotanchik-SB Just pushed in the change thanks!